mantle/kola/tests/misc/network.go

```diff
@@ -59,6 +61,15 @@ func init() {
 		Distros:     []string{"rhcos"},
 		Platforms:   []string{"qemu-unpriv"},
 	})
+	// This test follows the same network configuration used on https://github.com/RHsyseng/rhcos-slb
+	// with a slight change, where the MCO script is run from ignition: https://github.com/RHsyseng/rhcos-slb/blob/main/setup-ovs.sh.
+	// and we're using veth pairs instead of real nic when setting the bond
+	register.RegisterTest(&register.Test{
+		Run:         NetworkBondWithDhcp,
+		ClusterSize: 0,
+		Name:        "rhcos.network.bondWithDhcp",
+		Distros:     []string{"rhcos"},
```

Comment:
I think you need something like:

```
Tags:             []string{kola.NeedsInternetTag},
```

here. Otherwise by default we restrict internet access. See e.g. the Tang tests in `luks.go` as examples.

Reply:
Thanks! but I think that after the change i pushed it will no longer be need.